pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -542,3 +542,12 @@ func (k *Kubernetes) VolumesFrom(objects *[]runtime.Object, komposeObject kobjec
 	}
 	return nil
 }
+
+//Ensure the kubernetes objects are in a consistent order
+func SortedKeys(komposeObject kobject.KomposeObject) []string {
+	var sortedKeys []string
+	for name := range komposeObject.ServiceConfigs {
+		sortedKeys = append(sortedKeys, name)
+	}
+	return sortedKeys
```

Comment:
@surajssd Ya, as per the other part of the code: `sort.Strings(sortedKeys)` should be added :)